src/components/z-combobox/index.tsx

```diff
@@ -285,10 +284,9 @@ export class ZCombobox {
   private getComboboxA11yAttributes(): Record<string, string> {
     return {
       "role": "combobox",
-      "aria-label": this.label,
-      "aria-expanded": this.isopen ? "true" : "false",
-      "aria-activedescendant": this.isopen ? this.focusedItemId : "",
-      "aria-controls": `${this.inputid}_list`,
+      "html-aria-expanded": this.isopen ? "true" : "false",
```

Comment:
`getComboboxA11yAttributes` viene usato per aggiungere gli attributi anche su un tag nativo `span` in quel caso gli attributi custom `html-*`non hanno effetto.
 
`{!this.hassearch ? <span {...this.getComboboxA11yAttributes()} /> : null}`

ma ho un dubbio, mi chiedo se quella riga serva davvero?